### PR TITLE
Deprecate label_time_courses

### PR DESCRIPTION
**Pull request by teonlamont**
Addresses #1169.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
This must cause warnings during tests, right? All warnings in tests should be caught.

File: mne/label.py [hunk 1]
```diff
@@ -1036,6 +1037,8 @@ def split_label(label, parts=2, subject=None, subjects_dir=None,
     return labels
 
 
+@deprecated("'label_time_courses' will be removed in version 0.10, please use "
```
Comment (agramfort) [inline mne/label.py]: you should remove its use everywhere

see 

git grep label_time_courses

**Comment by teonlamont** [comment]
cool tip on git grep. very useful!

**Comment by teonlamont** [comment]
@Eric89GXL the only test that called label_time_courses was test_label_time_course

File: mne/__init__.py [hunk 2]
```diff
@@ -62,9 +62,9 @@
 from .epochs import Epochs, EpochsArray, read_epochs
 from .evoked import (Evoked, EvokedArray, read_evokeds, write_evokeds,
                      grand_average, combine_evoked)
-from .label import (label_time_courses, read_label, label_sign_flip,
-                    write_label, stc_to_label, grow_labels, Label, split_label,
-                    BiHemiLabel, read_labels_from_annot, write_labels_to_annot)
+from .label import (read_label, label_sign_flip, write_label, stc_to_label,
```
Comment (agramfort) [inline mne/__init__.py]: Keep it for backward compat

**Comment by teonlamont** [comment]
this is good to go

**Comment by agramfort** [comment]
please rebase then +1 for merge

**Comment by agramfort** [comment]
thx @teonlamont

**Pull request merged**